packages/story-editor/src/components/library/panes/pageTemplates/karma/savedPageTemplates.karma.js

```diff
@@ -118,6 +118,7 @@ describe('CUJ: Page Templates: Custom Saved Templates', () => {
       await fixture.events.hover(
         fixture.editor.library.pageTemplatesPane.pageTemplates[0]
       );
+      await fixture.events.sleep(100);
```

Comment:
Why is this needed?

Reply:
It seems that the button is not instantly actionable, if you lower that to 50 it will still fail, that's weird, given the fact that there is no fade-in animation etc.

Reply:
@littlemilkstudio I've tried `waitForIdle` with `requestIdleCallback` with no luck, same with removal of `queuePageImageGeneration` 🤷🏻

Reply:
ah bummer. probably not the issue then. thanks for exploring and reporting back here